### PR TITLE
Fix DD.parse for whole numbers (#260)

### PR DESCRIPTION
**Pull request by FObermaier**
Signed-off-by: Felix Obemaier <felix.obermaier@netcologne.de>

### CODE REVIEW

File: modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java [hunk 1]
```diff
@@ -88,6 +88,11 @@ private void checkSciNotation(DD x, String expectedStr) {
 	}
 
 	public void testParse() {
+		//checkParse("0", 0, 1e-32);
```
Comment (dr-jts) [inline modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java]: Any reason this is commented?

File: modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java [hunk 2]
```diff
@@ -88,6 +88,11 @@ private void checkSciNotation(DD x, String expectedStr) {
...
+		checkParse("1", 1, 1e-32);
+        checkParse("100", 100, 1e-32);
```
Comment (dr-jts) [inline modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java]: Would be nice to keep indentation consistent

**Comment by dr-jts** [comment]
Good catch - thanks for the patch.

File: modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java (hunk 1 above)
Comment (FObermaier) [reply modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java]: Problem calculating relerr (div/0), fixed that.

File: modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java (hunk 2 above)
Comment (FObermaier) [reply modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java]: Agreed

**Comment by dr-jts** [comment]
@FObermaier Thanks for the last commit. Can you amend it to have a sign-off?  Then I will merge.

**Comment by jnh5y** [comment]
@dr-jts we can use the 'Squash and merge' and then edit the commit message.  I can do that if you like.

**Comment by dr-jts** [comment]
@jnh5y Cool!  Please do...

**Comment by jnh5y** [comment]
@dr-jts @FObermaier thanks, and good job!

**Pull request merged**